docs/DeveloperGuide.md

```diff
@@ -115,7 +114,7 @@ The `PersonCard` part also has a reference to the `MainWindow` to provide improv
 
 Here's a (partial) class diagram of the `Logic` component:
 
-<img src="images/LogicClassDiagram.png" width="550"/>
+![Logic Class Diagram](images/LogicClassDiagram.png)
```

Comment:
I think you should keep the width here! same goes for line 135, Zac changed the width in a previous PR to make sure the fonts in all diagrams are more or less of a uniform size.

Reply:
@EssWhyy do you want to make a quick fix here? Then it's ok to be merged!